src/packages/context/context.py

```diff
@@ -10,22 +10,38 @@
 
 See LICENSE.md
 """
-from typing import Optional, Tuple, List, TYPE_CHECKING
+import typing
+from typing import Optional, Tuple, List, TYPE_CHECKING, Dict
 
-from config import config
+from src.config import CONFIG_MARKER
 from ..user import User
 
+import logging
+
+LOG = logging.getLogger(f"mecha.{__name__}")
 if TYPE_CHECKING:
     from src.mechaclient import MechaClient
 
-PREFIX = config['commands']['prefix']
+
+@CONFIG_MARKER
+def validate_config(data: Dict):
+    if not isinstance(data['commands']['prefix'], str):
```

Comment:
[Question] This value is validated twice, in Context and Rat_Command. Is that intentional?

Reply:
There is no restriction on what plugin may validate what portion of a configuration file.

Reply:
I'm not saying there should be, I just find it odd that it's validated **twice**. Seems like a good place for a "changed-in-one-place-but-not-the-other" error to appear.

Reply:
Ahh, I misunderstood.

Fair point, il look at it.

Reply:
validations in rat command have been removed. issue resolved